app/views/layouts/api_guidance/_header.html.erb

```diff
@@ -0,0 +1,15 @@
+<header class="govuk-header app-header" role="banner" data-module="govuk-header">
```

Comment:
could we use the following instead, and possibly not need the extra stylings for header?
```
<%=
  govuk_header(service_name: "NPQs data management API for lead providers") do |header|
  ...
  ```